support-frontend/stories/nav/Nav.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import { expect } from '@storybook/jest';
+import { userEvent, waitFor, within } from '@storybook/testing-library';
+import NavComponent from 'components/nav/nav';
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import {
+	AUDCountries,
+	Canada,
+	EURCountries,
+	GBPCountries,
+	International,
+	NZDCountries,
+	UnitedStates,
+} from 'helpers/internationalisation/countryGroup';
+
+export default {
+	title: 'SFD/Nav',
```

Comment:
This component should probably go under the 'Checkout' or 'Checkout Layout' category